src/main/java/appeng/core/localization/GuiText.java

```diff
@@ -55,9 +55,13 @@
 
 	inWorldFluix, inWorldPurificationCertus, inWorldPurificationNether,
 
-	inWorldPurificationFluix, inWorldSingularity, ChargedQuartz, OfSecondOutput,
+	inWorldPurificationFluix, inWorldSingularity, ChargedQuartz,
 
-	NoSecondOutput, Stores, Next, SelectAmount, Lumen, Empty,
+	NoSecondOutput,
+	OfSecondOutput,
+	MultipleOutputs,
```

Comment:
Fine for now, just wanted to ask in case the autoformatter triggered or so. But we can then slowly split them into multiple lines when editing it.